playground/TestShop/AppHost/ParameterExtensions.cs

```diff
@@ -0,0 +1,117 @@
+using System.Diagnostics;
+using System.Reflection;
+using Aspire.Hosting.Publishing;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.UserSecrets;
+using Microsoft.Extensions.Hosting;
+
+namespace Aspire.Hosting;
+
+internal static class ParameterExtensions
+{
+    /// <summary>
+    /// Creates a password parameter that in the development environment is generated once and stored in the user secrets store.
+    /// </summary>
+    /// <remarks>
+    /// The password is only stable when in the development environment and the application is running. In all other cases, the password is generated each time.
+    /// </remarks>
+    public static IResourceBuilder<ParameterResource> CreateStablePassword(this IDistributedApplicationBuilder builder, string name,
+        bool lower = true, bool upper = true, bool numeric = true, bool special = true,
+        int minLower = 0, int minUpper = 0, int minNumeric = 0, int minSpecial = 0)
+    {
+        ParameterDefault generatedPassword = new GenerateParameterDefault
```

Comment:
```suggestion
        ParameterDefault parameterDefault = new GenerateParameterDefault
```

Since sometimes this isn't a "generatedPassword" since it could come from user secrets.